10-JavaScript/fun-with-functions.md

```diff
@@ -1,34 +1,99 @@
-# Fun with Functions!
+# Intro to Functions!
 
 ## Learning Goals
-- Write and invoke some JavaScript functions
+
+- Intro to syntax (defining and calling), exercise, functions with parameters, exercise
+- Learn and practice the syntax for defining and calling/invoking functions in JavaScript
+- Learn and practice the syntax for using functions that have parameters
+
+<!-- - Write and invoke some JavaScript functions
 - Learn how JavaScript functions are different than Ruby methods
 - Distinguish between a functional _expression_ and a function _declaration_
-- Begin exploring what _higher order_ functions are
+- Begin exploring what _higher order_ functions are -->
+
+## Functions in JavaScript are like Methods in Ruby
+
+In JavaScript, we can use functions to define reusable behavior.
+
+This should _feel_ similar to methods in Ruby. They're not the same, but we can get into those details later. For now, we'll use functions to name and structure sections of code.
+
+Either way, the syntax for functions is a lot different in JavaScript than in Ruby.
+
+## Functions Are Objects, So We Assign Variables to Them
```

Comment:
```suggestion
## Functions Are Objects, So We Can Assign Them to Variables
```

Reply:
So I thought a lot about this... Idk, it's not _that_ important, but I really think of it as variable assignment that we assign variables to values, we don't assign values to variables........ idk idk but I would fight to keep as-is

Reply:
I initially parsed this as talking about being able to add fields to functions.